midi-item.cc:322:32: error: non-constant-expression cannot be narrowed from type 'Audio_text::Type' to 'uint8_t' (aka 'unsigned char') in initializer list [-Wc++11-narrowing]
  uint8_t text_code[] = {0xff, audio_->type_};
                               ^~~~~~~~~~~~~
midi-item.cc:322:32: note: insert an explicit cast to silence this issue
  uint8_t text_code[] = {0xff, audio_->type_};
                               ^~~~~~~~~~~~~
                               static_cast<uint8_t>( )

--- lily/include/audio-item.hh
+++ lily/include/audio-item.hh
@@ -110,17 +110,17 @@
 public:
   std::string type_string_;
   Direction dir_;
 };
 
 class Audio_text : public Audio_item
 {
 public:
-  enum Type
+  enum Type : uint8_t
   {
     TEXT = 1, COPYRIGHT, TRACK_NAME, INSTRUMENT_NAME, LYRIC,
     MARKER, CUE_POINT
   };
 
   Audio_text (Audio_text::Type type, const std::string &text_string);
 
   Type type_;
